apax/md/ase_calc.py

```diff
@@ -82,8 +82,10 @@ def make_ensemble(model):
     def ensemble(positions, Z, idx, box, offsets):
         results = model(positions, Z, idx, box, offsets)
         uncertainty = {k + "_uncertainty": jnp.std(v, axis=0) for k, v in results.items()}
+        ensemble = {k + "_ensemble": v for k, v in results.items()}
         results = {k: jnp.mean(v, axis=0) for k, v in results.items()}
         results.update(uncertainty)
+        results.update(ensemble)
```

Comment:
Does this mean, when using any kind of ensemble, we will have:

```
{energy: float, energy_ensemble: list[float], forces: ndarray[N, 3], forces_ensemble: ndarray[m, N, 3]}
```

and so on? I see where this can be very useful, but for the sake of performance it might be a good idea to make this toggleable? For e.g. saving to `h5` the amount of data will increase noticeably?

Reply:
Yes these are now available for all types of ensemble. passing arguments to the ASE calculator in IPS is a bit cumbersome at the moment. You always need a second node in addition to the training node (otherwise you have to retrain when changing the ASE options).
The inference speed should actually not be affected, but yes the storage size will increase significantly. Storing the ensemble predictions is relevant for some UQ metrics and reweighting.
I guess I can add an optional white list to the ASE calculator which filters the results by keys. same for jaxMD.
it seems like a cleaner solution to me to specify this in the h5 writer, but I can also implement it on the apax side.

Reply:
I guess it doesn't hurt to implement it here as well, but I agree that it is more useful on the `znh5md` side.

Reply:
I have to say that this would be really unergonomical when using the ASECalculator in IPSuite. I think models should return what they return and which properties are logged should be part of the trajectory writer. If I implement this as part of the ASECalc, it would mean that we train a model with a specified list of outputs and changing these properties would require a retraining or using a separate Node.
So I think this was a good addition to jaxMD (as it is  implemented in the trajectory writer), but for the ASECalculator it makes everything more complicated.